book/08-customizing-git/sections/attributes.asc

```diff
@@ -121,9 +121,9 @@ $ git config diff.exif.textconv exiftool
 
 If you replace an image in your project and run `git diff`, you see something like this:
 
-[source]
+[source,console]
 ----
-diff --git a/image.png b/image.png
+$ diff --git a/image.png b/image.png
```

Comment:
I was surprised to see Pygments does indeed support diff -- I'll give that a shot, thanks.